docs/tutorials/web/ep-webgpu.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Using WebGPU
+description: Using WebGPU
+parent: Web
+grand_parent: Tutorials
+has_children: false
+nav_order: 2
+---
+{::options toc_levels="2..4" /}
+
+# Using WebGPU Execution Provider
```

Comment:
It's also helpful to mention other env variables, like env.webgpu.adapter, env.webgpu.device, etc.